tests/foreman/cli/test_hostgroup.py

```diff
@@ -197,21 +198,21 @@ def test_positive_create_with_lifecycle_environment(self):
 
         @Assert: Hostgroup should be created and has lifecycle env assigned
 
-        @BZ: 1309107
+        @BZ: 1359694
         """
         org = make_org()
         lc_env = make_lifecycle_environment({'organization-id': org['id']})
         hostgroup = make_hostgroup({
             'lifecycle-environment': lc_env['name'],
             'organization-ids': org['id'],
-            'lifecycle-environment-organization-id': org['id'],
         })
+        print hostgroup
```

Comment:
ah.. Yeah.. Forgot to remove.. found another issue and filed bz1360136. will update the PR